zookeeper/include/zk_nn_client.h

```diff
@@ -180,11 +217,9 @@ class ZkNnClient : public ZkClientCommon {
 		const int IS_FILE = 2;
 		const int IS_DIR = 1;
 		// TODO: Should eventually be read from a conf file
-		const int ACK_TIMEOUT = 60; // 60 second timeout when waiting for replication acknowledgements
-
+        const int ACK_TIMEOUT = 600000; // in millisecons, 10 minute timeout when waiting for replication acknowledgements
```

Comment:
Architecture document states that the items of the wait_for_acks queue have znode name: <block_uuid> and payload: {timeout : long, replication_factor : int}

Check where things are being added to wait_for_acks and ensure that it follows this convention. Then update check_acks to read the time out value from this znode in ZK, rather than using this constant.

Assigned to @prb2